LDAR_Sim/src/default_parameters/virtual_world_default.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.0"
+parameter_level: "virtual_world"
+weather_file: "weather_alberta.nc"
+weather_is_hourly: True # True/False
+infrastructure_file: "facilities_alberta.csv"
+site_samples: 500
+subtype_times_file: "_placeholder_str_"
+subtype_file: "_placeholder_str_"
+consider_weather: False # True/False
+repair_delay:
```

Comment:
This might be a Program Param

Reply:
Ultimately I think repair delay, repair cost and verification costs should be tied to emissions in some way, but after review I agree having them as program parameters makes the most sense for now.

Reply:
Closing the circle here: Previously functionality does not allow for repair times to be set on a per program basis. This will be addressed in a separate feature/ bugfix PR. The economics parameters will be moved back to the program level.